packages/sdk/Dockerfile

```diff
@@ -3,27 +3,34 @@ ARG BASE_IMAGE
 ARG SERVER_MANAGER_REGISTRY
 ARG SERVER_MANAGER_ORG
 ARG SERVER_MANAGER_VERSION
-ARG LINUX_VERSION
+ARG CARTESI_IMAGE_KERNEL_VERSION
 ARG LINUX_KERNEL_VERSION
-ARG ROM_VERSION
 
 FROM ${BASE_IMAGE} as builder
 
 WORKDIR /usr/local/src
+ARG DEBIAN_FRONTEND=noninteractive
 RUN <<EOF
 set -e
 apt-get update
-apt-get install -y --no-install-recommends autoconf automake build-essential ca-certificates curl libarchive-dev libtool
+apt-get install -y --no-install-recommends \
+    autoconf \
+    automake \
+    build-essential \
+    ca-certificates \
+    curl \
+    libarchive-dev \
+    libtool
 rm -rf /var/lib/apt/lists/*
 EOF
 
 FROM builder as genext2fs
 
+# main -> a7abe9ecaeb376babc4148c1122dfb9103539d2c
```

Comment:
Let's use the tagged commit

`v1.5.3 -> 1c1992153eb77cc59a8447c631b141c955d37c2a `

Reply:
The tagged commit does not have the latest fix @mpolitzer did

Reply:
maybe we should ask him to make a release

Reply:
but ok, if it doesn't have any other change, the tag will eventually point to the same commit

Reply:
anyway, the branch name on @mpolitzer's fork is `cartesi`.

Reply:
I don't know if he has more commits to do to make it "1.5.4". Isn't there a `--version` that needs a change?

Reply:
probably https://github.com/cartesi/genext2fs/blob/cartesi/configure.ac#L5

Reply:
https://github.com/cartesi/genext2fs/commit/1c1992153eb77cc59a8447c631b141c955d37c2a